virtcontainers/api_test.go

```diff
@@ -534,7 +338,10 @@ func TestStartSandboxKataAgentSuccessful(t *testing.T) {
 	pImpl, ok := p.(*Sandbox)
 	assert.True(t, ok)
 
-	bindUnmountAllRootfs(ctx, defaultSharedDir, pImpl)
+	// TODO: defaultSharedDir is a hyper var = /run/hyper/shared/sandboxes
+	// do we need to unmount sandboxes and containers?
+	bindUnmountAllRootfs(ctx, testDir, pImpl)
```

Comment:
@WeiZhang555 your comment reminded me I needed ask about this, thank you! 
Two test cases were calling `bindUnmountAllRootfs` with the defaultSharedDir as an argument, however neither test had anything to do with hyperstart (since `defaultSharedDir = /run/hyper/shared/sandboxes/`). I wasn't sure what to do about that situation; whether to bindUnmount the KataHostSharedDir, testDir, or some other variable instead.

Reply:
I'm not sure either, I think it should be `kataHostSharedDir`, but I wonder if this is really necessary, maybe `bindUnmountAllRootfs` can be removed.